ci-operator/step-registry/aws/deprovision/secureboot/aws-deprovision-secureboot-commands.sh

```
@@ -0,0 +1,27 @@
+set -o nounset
+set -o errexit
+set -o pipefail
+
+RESOURCES_FILE="${SHARED_DIR}/secureboot-resources.txt"
+
+if [[ ! -f "${RESOURCES_FILE}" ]]; then
+  echo "No Secure Boot resources to clean up"
+  exit 0
+fi
+
+source "${RESOURCES_FILE}"
```

Shellcheck (0.11.0)

[warning] ShellCheck can't follow non-constant source. Use a directive to specify location.

(SC1090)